INSTALL

```diff
@@ -0,0 +1,34 @@
+There are two ways to install Johnny which depends on your platform.
+1. Use our easy installer (Windows and OS X only)
+OR
+2. Download source and build johnny (all platforms)
+
+1. Easy-install : use our installer (Windows and OS X only)
+-----------------------------------
+Simply visit http://openwall.info/wiki/john/johnny to get the latest installer
+for your platform. It comes bundled with a working John The Ripper binary.
+So you have everything ready to start cracking passwords.
+
+2. Getting the source code and compiling on Unix/Linux, Windows and OS X
```

Comment:
Separate the build instructions per platform family, like in the Tarsnap example I gave you.